src/web_app_skeleton/src/shards.cr.ecr

```diff
@@ -3,8 +3,9 @@ require "lucky_env"
 LuckyEnv.load?(".env")
 
 # Require your shards here
-require "avram"
 require "lucky"
+require "avram"
+require "avram/lucky"
```

Comment:
Could you make "avram/lucky" require the base avram library?

Reply:
It already does, but I went the more explicit route